.travis.yml

```diff
@@ -10,7 +10,7 @@ sudo: false
 
 cache:
   directories:
-    - $HOME/.composer/cache/files
+    - $HOME/.composer
```

Comment:
Hmm, caching the whole composer directory looks dangerous to me. Why this is needed?

Reply:
to have `prestissimo` cached. whole point of this PR.

I am caching whole composer for 3+ years, never had any issues because of that.
Why you find it dangeroues ?

Reply:
I find it dangerous because we might have not wanted cached file and I already add not wanted dependencies issues before.

But it was a long time ago, let's get a try.

Reply:
well, you can put anything in cache, what would be the outcome deps depends only on composer.* files, not caceh